app/views/sessions/_form.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="field" >
+	<%= f.label :name %>
+	<%= f.text_field :name %><br><br>
```

Comment:
Instead of using `<br><br>`, consider using styles appropriately to style the form to look how you want it to look.
Adding the break tags means adding presentation logic into your html markup.